BackEndFlask/Functions/test_files/test_teamBulkUpload.py

```diff
@@ -0,0 +1,224 @@
+from Functions.customExceptions import *
+from Functions.teamBulkUpload import team_bulk_upload
+from population_functions import *
+import os
+
+def retrieveFilePath(fileName):
+    return os.getcwd() + os.path.join(os.path.sep, "Functions") + os.path.join(os.path.sep, "sample_files")+ os.path.join(os.path.sep, "teamBulkUpload-files") + os.path.join(os.path.sep, fileName)
+
+# test_wrong_extention_error
+#   - ensures that student_and_team_to_db
+#       - returns an error if given the incorrect file type
+def test_should_fail_with_wrong_extention_error(flask_app_mock):
```

Comment:
Is this function supposed to be mostly commented out? The assert true seems off.